packages/aws-cdk-lib/aws-ecs-patterns/lib/fargate/scheduled-fargate-task.ts

```diff
@@ -97,6 +98,9 @@ export class ScheduledFargateTask extends ScheduledTaskBase {
     if (props.memoryLimitMiB) {
       Annotations.of(this).addWarningV2('@aws-cdk/aws-ecs-patterns:propertyIgnored', 'Property \'memoryLimitMiB\' is ignored, use \'scheduledFargateTaskImageOptions.memoryLimitMiB\' instead.');
     }
+    if (props.ephemeralStorageGiB) {
+      Annotations.of(this).addWarningV2('@aws-cdk/aws-ecs-patterns:propertyIgnored', 'Property \'ephemeralStorageGiB\' is ignored, use \'scheduledFargateTaskImageOptions.ephemeralStorageGiB\' instead.');
+    }
```

Comment:
Not familiar with this file, whats the purpose of this warning?

Reply:
Thank you. From the issue description https://github.com/aws/aws-cdk/issues/18105: `aws-ecs-patterns is inconsistent in how it exposes some configuration options. FargateScheduledTask puts the cpu/memory options inside FargateTaskImageOptions`. 

Those warnings are used to tell users not pass `cpu`, `memory`, and `storage` by props. They should put them in `FargateTaskImageOptions` instead.

Reply:
I was going to ask to move the property to the top level but I think it makes more sense to have it with cpu and memoryLimitMiB